### PR TITLE
[4.0] css changes in voting plugin

### PR DESCRIPTION
**Pull request by rjharishabh**
Pull Request for Issue #32908.

### Actual result BEFORE applying this Pull Request

before voting

### Expected result AFTER applying this Pull Request

vote

### Documentation Changes Required
No

### CODE REVIEW

**Comment by brianteeman** [comment]
why are you changing the height?

**Comment by rjharishabh** [comment]
> why are you changing the height?

for mobile users

**Comment by infograf768** [comment]
@rjharishabh 
Have y7ou tested https://github.com/joomla/joomla-cms/pull/31098

**Comment by rjharishabh** [comment]
> @rjharishabh
> Have y7ou tested #31098

No

**Comment by brianteeman** [comment]
That was a hint that you probably should

**Comment by rjharishabh** [comment]
> That was a hint that you probably should

yeah sure, I will test

**Comment by rjharishabh** [comment]
please test this PR @brianteeman @infograf768

**Comment by rjharishabh** [comment]
I have tested PR #31098 

Some error occurred

File: plugins/content/vote/tmpl/vote.php [hunk 1]
```diff
@@ -39,8 +39,8 @@
 <form method="post" action="<?php echo htmlspecialchars($uri->toString(), ENT_COMPAT, 'UTF-8'); ?>" class="form-inline">
 	<span class="content_vote">
 		<label class="visually-hidden" for="content_vote_<?php echo (int) $row->id; ?>"><?php echo Text::_('PLG_VOTE_LABEL'); ?></label>
-		<?php echo HTMLHelper::_('select.genericlist', $options, 'user_rating', 'class="form-control form-control-sm"', 'value', 'text', '5', 'content_vote_' . (int) $row->id); ?>
-		&#160;<input class="btn btn-sm btn-primary" type="submit" name="submit_vote" value="<?php echo Text::_('PLG_VOTE_RATE'); ?>">
+		<?php echo HTMLHelper::_('select.genericlist', $options, 'user_rating', 'class="form-control form-control-sm w-auto d-inline-block"', 'value', 'text', '5', 'content_vote_' . (int) $row->id); ?>
+		<input class="btn  btn-sm btn-primary ml-1" type="submit" name="submit_vote" value="<?php echo Text::_('PLG_VOTE_RATE'); ?>">
```
Comment (Quy) [inline plugins/content/vote/tmpl/vote.php]:
```suggestion
		<input class="btn btn-sm btn-primary ml-1" type="submit" name="submit_vote" value="<?php echo Text::_('PLG_VOTE_RATE'); ?>">
```

Comment (rjharishabh) [reply plugins/content/vote/tmpl/vote.php]: Thanks

**Comment by rjharishabh** [comment]
@Quy I think it's fine now

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 859039e3d3056b32cdd42a525e34fe2cd3d74e57<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32919">issues.joomla.org/tracker/joomla-cms/32919</a>.</sub>

**Comment by rjharishabh** [comment]
@infograf768  Thnx for testing

**Comment by ceford** [comment]
I tested this and it works as described but it seems it will be redundant when #31098 gets merged - and it has a drop-down chevron,<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32919">issues.joomla.org/tracker/joomla-cms/32919</a>.</sub>

**Comment by rjharishabh** [comment]
I think it looks better now

**Comment by sandewt** [comment]
I have tested this item :red_circle: unsuccessfully on efe66e7e23a39fdab597ea0d8b04fe958503e752<br /><br />Patch doesn't work in responsive mode<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32919">issues.joomla.org/tracker/joomla-cms/32919</a>.</sub>

**Comment by sandewt** [comment]
The patch doesn't work in the responsive mode: see image


screen shot 2021-03-31 at 09 50 05


You can solve this with the following code: `'style' => 'width: 120px;'`
This results in:
```
<?php echo HTMLHelper::_('select.genericlist', $options, 'user_rating', array('class' => 'form-select form-select-sm', 'style' => 'width: 120px;'), 'value', 'text', '5', 'content_vote_' . (int) $row->id); ?>

```
issue-2
[EDIT]

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32919">issues.joomla.org/tracker/joomla-cms/32919</a>.</sub>

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on ee406f163fdef2e159b119dddf018618ef6c2c7d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32919">issues.joomla.org/tracker/joomla-cms/32919</a>.</sub>

**Comment by rjharishabh** [comment]
Thanks for testing @sandewt

**Comment by rjharishabh** [comment]
please test this PR @infograf768 and @ceford again
there are some changes

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on ee406f163fdef2e159b119dddf018618ef6c2c7d<br /><br />Works in wide and narrow screen modes. But no css changes in diff and Vote is below <Prev button - a separate issue.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32919">issues.joomla.org/tracker/joomla-cms/32919</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32919">issues.joomla.org/tracker/joomla-cms/32919</a>.</sub>

**Comment by rjharishabh** [comment]
> I have tested this item ✅ successfully on [ee406f1](https://github.com/joomla/joomla-cms/commit/ee406f163fdef2e159b119dddf018618ef6c2c7d)
> 
> Works in wide and narrow screen modes. But no css changes in diff and Vote is below <Prev button - a separate issue.
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/32919](https://issues.joomla.org/tracker/joomla-cms/32919).

PR #32914

**Comment by infograf768** [comment]
imho, I prefer https://github.com/joomla/joomla-cms/pull/31098 using svg.

**Comment by drmenzelit** [comment]
Thanks @rjharishabh for your first contribution!
The select field and the button are not aligned correctly, but that can be improved on another PR
grafik

**Pull request merged**